scripts/install

```diff
@@ -107,6 +107,14 @@ ynh_script_progression --message="Starting a systemd service..." --weight=3
 # Start a systemd service
 ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="Listening on"
 
+#=================================================
+# SETUP SSOWAT
+#=================================================
+ynh_script_progression --message="Configuring permissions..." --weight=3
+
+ynh_permission_url --permission="main" --auth_header="false"
+ynh_permission_url --permission="api" --auth_header="false"
+
```

Comment:
This was the old way to change permissions. Now all is done in manifest.toml :)
```suggestion
```

Reply:
(didn’t find the way to suggest on manifest oops)

Reply:
Moved the auth_header changes to the manifest. Should be good to go now.